source/cloud-security/amazon/services/prerequisites/dependencies.rst

```diff
@@ -53,9 +53,24 @@ The required modules can be installed with Pip, the Python package manager. Most
 
 It is recommended to use a pip version greater than or equal to 19.3 to ease the installation of the required dependencies.
 
-.. code-block:: console
+.. tabs::
+
+   .. group-tab:: Python 3.7 - 3.10
+
+      .. code-block:: console
+
+         # pip3 install --upgrade pip
+
+   .. group-tab:: Python 3.11
+
+      .. code-block:: console
+
+         # pip3 install --upgrade pip --break-system-packages
+   
+      .. note::
+         
+         The ``--break-system-packages`` parameter is required to make the install on the default externally managed environment (more information on the `PEP 668 description <https://peps.python.org/pep-0668/>`_). To avoid it, the command can be executed inside a virtual environment but this would require a modification on the ``aws-s3`` script shebang to use the environment's interpreter. 
```

Comment:
```suggestion
         This command modifies the default externally managed Python environment. See the `PEP 668 <https://peps.python.org/pep-0668/>`__ description for more information.
         
         To prevent the modification, you can run ``pip3 install --upgrade pip`` within a virtual environment. You must update the ``aws-s3`` script shebang with your virtual environment interpreter, for example, ``#!/path/to/your/virtual/environment/python``.
```